.drone.yml

```diff
@@ -8,18 +8,17 @@ platform:
 
 steps:
 - name: mirror-images
-  image: docker
+  image: rancher/dapper:v0.4.2
```

Comment:
Is there a reason you've pinned to `v0.4.2` instead of one of the newer dapper versions?

Reply:
It appeared to be what we were using for other projects. I can update.